docs/content/manual/release-specific/v1.1.2/instance-manager-streaming-connection-recovery.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Test Instance Manager Streaming Connection Recovery
+---
+
+## Related issue
+https://github.com/longhorn/longhorn/issues/2561
+
+## Test Step
+**Given** A cluster with Longhorn 
+
```

Comment:
I think, we should add a step here - `Create a volume and attach it to a pod`. Now we can try killing the connection of IM and make sure the volume comes back online healthy.